files/setup.sh

```diff
@@ -54,6 +56,17 @@ else
         echo
 	fi
 
+	# Determine Knative deployment model
+	if [ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]; then
+		if [ ! -z ${KNATIVE_HOST} ]; then
+			KNATIVE_DEPLOYMENT_TYPE="external"
+		else
+			KNATIVE_DEPLOYMENT_TYPE="embedded"
+		fi
+	else
+		KNATIVE_DEPLOYMENT_TYPE="na"
```

Comment:
What is this doing at all? I don't see any fallback/error logic around this value.

Reply:
The fallback is `na` to ensure that the shell scripts don't just bail out because this variable wasn't defined for non-Knative setup. 

The reason this is being set is that I can then distinguish between Knative (Embedded) vs Knative (External) Broker configuration since both have slight configuration differences